rskj-core/src/test/java/co/rsk/config/BridgeConstantsTest.java

```diff
@@ -122,4 +123,22 @@ void test_getMinimumPeginTxValue(BridgeConstants bridgeConstants, boolean isRSKI
             assertEquals(bridgeConstants.legacyMinimumPeginTxValue, minimumPeginTxValue);
         }
     }
+
+    private static Stream<Arguments> getBtcHeightWhenPegoutTxIndexActivatesArgProvider() {
+        return Stream.of(
+            Arguments.of(BridgeMainNetConstants.getInstance(), 100),
+            Arguments.of(BridgeTestNetConstants.getInstance(), 150),
+            Arguments.of(BridgeRegTestConstants.getInstance(), 250)
+        );
+    }
+
+    @ParameterizedTest()
+    @MethodSource("getBtcHeightWhenPegoutTxIndexActivatesArgProvider")
+    void test_getBtcHeightWhenPegoutTxIndexActivates(BridgeConstants bridgeConstants, int expectedValue){
+        // Act
+        int btcHeightWhenPegoutTxIndexActivates = bridgeConstants.getBtcHeightWhenPegoutTxIndexActivates();
+
+        // assert
+        Assertions.assertEquals(btcHeightWhenPegoutTxIndexActivates, expectedValue);
```

Comment:
```suggestion
        Assertions.assertEquals(expectedValue, btcHeightWhenPegoutTxIndexActivates);
```
expected value goes first :)

Reply:
Done.